pkg/cli/image/extract/extract.go

```diff
@@ -201,7 +201,7 @@ func parseMappings(images, paths, files []string, requireEmpty bool) ([]Mapping,
 			}
 			mappings = append(mappings, Mapping{
 				Image: image,
-				From:  strings.TrimPrefix(arg, "/"),
+				From:  path.Clean(arg),
```

Comment:
I'd at least start with filepath, which works fine on any OS. path is unix only.

Reply:
Isn't this the path _in the container_ - so it is going to be a UNIX filesystem? Nobody is doing e.g. `C://` to refer to something in the container FS, are they?

Reply:
What when the container is windows? Obviously it's not here, but I'd prefer to fix it now, rather then later. It's not that hard :wink:

Reply:
Ah, gotcha